src/js/components/CinnamonRoleUsersTable.vue

```diff
@@ -20,6 +21,7 @@
                     <tr v-for="user in users">
                         <td style="vertical-align: middle;" v-text="user.attributes.name"></td>
                         <td style="vertical-align: middle;"><code v-text="user.attributes.email"></code></td>
+                        <td style="vertical-align: middle;"><code v-text="user.relationships.role.data.attributes.name"></code></td>
```

Comment:
This same thing can be achieved using a getter method. Something like `getRoleForUser(user)`. Then you can find the role matching the users `role_id` and return that.